key/aws-kms-key.json

```diff
@@ -104,12 +104,14 @@
     },
     "KeyId": {
       "type": "string"
+    },
+    "BypassPolicyLockoutSafetyCheck": {
+      "description": "Does not skip the key policy lockout safety check.",
```

Comment:
Have Alie / Karen review this. I think the current text is a bit confusing, it sounds like true means it will NOT skip the lockout check.

Reply:
Is this something we can copy from public docs ?

Reply:
Yes. That's what I asked Alie. I'll add the description from the public docs.